backend/layers/processing/schema_migration.py

```diff
@@ -18,27 +19,30 @@
     DatasetVersionId,
 )
 from backend.layers.processing import logger
+from backend.layers.processing.process_logic import ProcessingLogic
 from backend.layers.thirdparty.step_function_provider import StepFunctionProvider
 
 logger.configure_logging(level=logging.INFO)
 
 
-class SchemaMigrate:
+class SchemaMigrate(ProcessingLogic):
     def __init__(self, business_logic: BusinessLogic):
+        # TODO: remove dependecies on ProcessingLogic
```

Comment:
is this TODO for the future?

Reply:
yes, in the event we want to completely decouple from ProcessnigLogic.